docs/api/document.md

```diff
@@ -189,29 +191,65 @@ const server = http.createServer(async (req, res) => {
 
 <!--END_DOCUSAURUS_CODE_TABS-->
 
+## Assets
+
+On the [`HttpIncoming`](incoming.md) object which is passed on to the document
+template one can find an array of `AssetCSS` objects on the `.css` property and
+an array of `AssetJS` objects on the `.js` property . These properties hold the
+assets of a podlet or a layout. In a layout they can additionally to the layouts
+assets, hold the assets of the requested podlets.
```

Comment:
In a layout they can hold the assets of the requested podlets in addition to the assets of the layout itself.